sirepo/pkcli/job_agent.py

```diff
@@ -751,16 +751,7 @@ async def _prepare_simulation(self):
         await c._await_exit()
 
     def _sbatch_script(self):
-        def _assert_project():
-            p = self.msg.sbatchProject
-            if not p:
-                return ""
-            o = subprocess.check_output(["hpssquota"], text=True)
-            assert re.search(r"^[-\w]+$", p), f"invalid NERSC project={p}"
-            assert re.search(
-                r"{}\s+\d+\.".format(p), o
-            ), f"sbatchProject={p} is invalid. hpssquota={o}"
-            return f"#SBATCH --account={p}"
+        import sirepo.nersc
```

Comment:
from sirepo import nersc when in a function. However, it doesn't need to be in a function, since this is a simple module.

Reply:
[6fb1a0e](https://github.com/radiasoft/sirepo/pull/5493/commits/6fb1a0edbd1d45106f8b767799ec29c8fef45527)